content/docs/releases/release-notes/release-notes-1.14.md

```diff
@@ -14,12 +14,25 @@ support for creating [CA certificates with "Name Constraints" and "Authority Inf
 >
 > Read [The cert-manager Command Line Tool (cmctl) page](../../reference/cmctl.md) to learn more.
 
+## `v1.14.2`
+
+> 📢 When upgrading to cert-manager release 1.14, please skip `v1.14.0` and `v1.14.1` and install this patch version instead.
+
+### Changes since `v1.14.1`
+
+#### Bug or Regression
+
+- BUGFIX: cert-manager CA and SelfSigned issuers incorrectly copied the critical flag from the CSR instead of re-calculating that field themselves. (#6727, @jetstack-bot)
+- Helm: Fix a bug in the logic that differentiates between 0 and an empty value. (#6729, @jetstack-bot)
```

Comment:
The PR and authors should be markdown links.
> Replace the GitHub issue numbers and GitHub handles (e.g., #1234 or @maelvls) with actual links using the following command:
https://cert-manager.io/docs/contributing/release-process/#minor-releases